src/InsertCapablePdoTrait.php

```diff
@@ -26,7 +26,11 @@
      *
      * @since [*next-version*]
      *
-     * @param array[]|ArrayAccess[]|stdClass[]|ContainerInterface[]|Traversable $records A list of records to insert.
+     * @param array|Traversable $records A list of records data containers. Accepted container types are:
```

Comment:
This is not accurate. The list can also be an `stdClass`. Also, the previous doc used to reflect the types in a structured way, and therefore allowed static analysis. The new doc will not allow static analysis tools to understand whether the list contains valid items.